tests/acceptance/expected-failures-with-oc10-server-openid-login.md

```diff
@@ -13,7 +13,3 @@ Other free text and markdown formatting can be used elsewhere in the document if
 
 ### [Browser session deleted user should not be valid for newly created user of same name](https://github.com/owncloud/ocis/issues/904)
 -   [webUILogin/openidLogin.feature:74](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUILogin/openidLogin.feature#L74)
-
-### [sorting for files lists needs to be reimplemented](https://github.com/owncloud/ocis/issues/1179)
--   [webUIFilesList/sort.feature:51](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesList/sort.feature#L51)
--   [webUIFilesList/sort.feature:72](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesList/sort.feature#L72)
```

Comment:
These line numbers don't point to any specific scenario but rather blank lines and data table
https://github.com/owncloud/web/blob/aa21f3726beb10e154afa72c0342b39a451c57a2/tests/acceptance/features/webUIFilesList/sort.feature#L51
https://github.com/owncloud/web/blob/aa21f3726beb10e154afa72c0342b39a451c57a2/tests/acceptance/features/webUIFilesList/sort.feature#L72